ios/theoAds/THEOplayerRCTTHEOAdsEventAdapter.swift

```diff
@@ -39,7 +40,8 @@ class THEOplayerRCTTHEOadsEventAdapter {
         if let duration = interstitial.duration {
             interstitialData[PROP_INTERSTITIAL_DURATION] = duration
         }
-        
+        interstitialData[PROP_AD_TAG_PARAMETERS] = fromInterstitialAdTagParameters(interstitial.adTagParameters)
```

Comment:
This is at the moment of writing incorrect as there is still a PR open to move `adTagParameters` to the Interstitial level while previously it was only for `adBreakInterstitial`. Since it's best to align across SDKs I added it already here in anticipation of it being there.

Also I followed the structure like for other maps where it is converted into [String:Any]. I do not know if this is actually necessary or not or why this is being done so please check and correct me 🙏